genet/validate/network.py

```diff
@@ -35,3 +36,33 @@ def describe_graph_connectivity(G):
     # find number of connected subgraphs
     dict_to_return['number_of_connected_subgraphs'] = len(find_connected_subgraphs(G))
     return dict_to_return
+
+
+def zero_value(value):
+    return value in {0, '0', '0.0'}
+
+
+def negative_value(value):
+    if isinstance(value, str):
+        return '-' in value
```

Comment:
I could pass the string `non-integer` and be told it is negative. Do we want that? Might be safer to parse to int and perform a numerical check. There is also the string function `startswith()`, which seems a better fit than `in` here.

Reply:
ah yes, another good point